src/oss/langchain-messages.mdx

```diff
@@ -2,14 +2,488 @@
 title: Messages
 ---
 
+Messages are the fundamental unit of communication in LangChain chat models. They represent the input and output of conversations, carrying both content and metadata that enables rich, multimodal interactions.
+
+TODO link out to chat model conceptual guide
```

Comment:
what will go in here?

context: we are trying to decide how to structure conceptual guides vs more technical guides